api/src/convert/dcap_evidence.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2023 The MobileCoin Foundation
+
+//! Convert to/from external::DcapEvidence
+
+use crate::{external, ConversionError};
+use mc_attest_verifier_types::DcapEvidence;
+use mc_sgx_dcap_types::{Collateral, Quote3};
+
+impl From<&DcapEvidence> for external::DcapEvidence {
+    fn from(src: &DcapEvidence) -> Self {
+        let mut dst = Self::new();
+
+        let quote = mc_util_serial::serialize(&src.quote).expect("Invalid Dcap Evidence");
```

Comment:
This feels a bit funky to me, but I'm not sure what else we can do here without getting all the SGX DCAP structs to have a proto version...
Doing it like this pretty much means only Rust code can ever do anything with this field. Not necessarily a problem, but I think we don't have any places where that's case (I might be wrong).

Reply:
Having all the SGX stuff implement prost (maybe with a feature flag) would be nice... I know it's a non-trivial amount of work but that's what we've done in many other places :/

Reply:
I think I'll pull it in and do it for quote and collateral.  We can come back and fill any others we need.

Reply:
@eranrund I created, https://mobilecoin-eng.atlassian.net/browse/CORE-58
My thought being that while this implementation isn't ideal as it stands, it can move us forward to getting more of the DCAP implementation in while `CORE-58` can be done anytime before releasing 6.0.0

Not sure your thoughts there?  I don't like putting something in we know we don't want in the final code, but one thing we've struggled with is ensuring we have tasks that others can work on. By not blocking subsequent work we ensure whoever is available can take up the prost/protobuf re-work.

Reply:
I'm okay with it if you trust us to not forget it. In many cases this is an easy decision (to postpone so we can keep making progress), but in this particular case the cost of forgetting to fix this would mean another block version and more annoying code to keep around forever. So I guess I'm fine with this approach if you trust the release process/your own process to ensure this is addressed before we go live.

Reply:
I'm going to sit on this and hope I have an epiphany over the weekend.
I agree that forgetting this fix in the 6.0.0 release would be bad, I want to see how far I get on my task to convert over to DCAP quotes as I think that will inform what other outstanding tasks I've missed, thus indicating if where people should focus.